workflow/scripts/featurecounts_deseq2.R

```diff
@@ -285,16 +285,30 @@ if (file.exists(ResultsFile) == FALSE) {
 
                 ## WRITE RESULTS FILE
                 if (MIN_FDR == 0.01) {
-                    CompResultsFile <- snakemake@output[["FDR_1_perc_res"]]
-                    CompBEDFile <- snakemake@output[["FDR_1_perc_bed"]]
-                    MAplotFile <- snakemake@output[["plot_FDR_1_perc_MA"]]  # AVI: added to create separate pdf files
-                    VolcanoPlotFile <- snakemake@output[["plot_FDR_1_perc_volcano"]]  # AVI: added to create separate pdf files
+                    # AVI: dynamically file name extensions added
+                    dirs <- c(snakemake@output[["FDR_1_perc_res"]],snakemake@output[["FDR_1_perc_bed"]],snakemake@output[["plot_FDR_1_perc_MA"]],
+                              snakemake@output[["plot_FDR_1_perc_volcano"]],snakemake@output[["FDR_5_perc_res"]],snakemake@output[["FDR_5_perc_bed"]],
+                              snakemake@output[["plot_FDR_5_perc_MA"]],snakemake@output[["plot_FDR_5_perc_volcano"]],snakemake@output[["plot_sample_corr_heatmap"]],
+                              snakemake@output[["plot_scatter"]])
+                    for (dir in dirs) {
+                        if (file.exists(dir) == FALSE) {
+                            dir.create(dir,recursive=TRUE)
+                        }
+                    }
+
+                    CompResultsFile <- paste(snakemake@output[["FDR_1_perc_res"]], "/", snakemake@params[["antibody"]], ".", CompPrefix, ".FDR_0.01_results.txt", sep="")
```

Comment:
As Snakemake can theoretically be run on Windows, we should try to work with file-system agnostic path names, here. I would assume that Snakemake ensures that what's in `snakemake@output[["whatever"]]` is in the correct format, to here we would make this:
```suggestion
                    CompResultsFile <- file.path(snakemake@output[["FDR_1_perc_res"]], paste(snakemake@params[["antibody"]], ".", CompPrefix, ".FDR_0.01_results.txt", sep="") )
```
Accordingly for the filenames below.